src/Application/Config.cpp

```diff
@@ -0,0 +1,84 @@
+//   Copyright 2020 Roddie Kieley
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+#include "Config.h"
+#include "yaml-cpp/node/node.h"
+#include "yaml-cpp/node/parse.h"
+#include "../Logging/loguru.hpp"
+#include <cstring>
+
+// Define the static Singleton pointer
+Config* Config::inst_ = nullptr;
+
+Config::Config() {
```

Comment:
With the constructor we should maintain the current c++ style in use:

// Constructor(s)
Pod::Pod(_Dependencies& theDependencies) :
    m_pBulletTimer(new Rock2D::Timer(500)),
    AEntity(theDependencies.UUID,
            (uint64_t)MakeT<uint64_t>((uint32_t)AEntity::POD, s_ui32Count), theDependencies.pB2DEntity),
    m_i16GroupCount(0)
{

}

Reply:
This is the only change I didn't implement.   I had to look up what this form was.  I learned that initializtion lists are useful for when initializing const member variables or parent class variables.  Since neither of these is true for these members, I see no difference in using an initialization list vs initializing these in the body of the constructor.  only that initializing them in the body is much much more readable.

Reply:
Actually never mind!  I just moved the assignment of the default values to the header file and removed the constructor all together.  I like that much better. So this is done!